packages/cli-repl/src/arg-parser.ts

```diff
@@ -73,7 +73,7 @@ const OPTIONS = {
       return true;
     }
     throw new Error(
-      `  ${clr(i18n.__(UNKNOWN), ['red', 'bold'])} ${clr(parameter, 'bold')}
+      `  ${clr(i18n.__(UNKNOWN), ['red', 'bold'], { colors: true })} ${clr(parameter, 'bold', { colors: true })}
```

Comment:
why here is ok to set `colors= true`?

Reply:
It’s not really, but we’re also not explicitly printing anything here … if I understand correctly, this gets passed down to the Node.js error handler itself

So a “clean” solution would be e.g. creating an error object here that has a colored and a non-colored message, and then handling that at the `parseCliArgs` call side in `src/run.ts` and deciding what to print from there … but I’m not sure if that’s worth the extra complexity? Especially that we’re targeting scripted usage with the color-on/color-off handling, and wouldn’t typically contain argument name misspellings (or not for a long time :))

Reply:
Fwiw, I’ve made this use the same thing as above, i.e. basically assume that this ends up on stderr (which I think is accurate)